lib/send_with_us/api.rb

```diff
@@ -277,10 +277,24 @@ def customer_remove_from_group(email_address, group_id)
       SendWithUs::ApiRequest.new(@configuration).delete(endpoint)
     end
 
-    def logs(options = {})
-      endpoint = "logs"
+    def customer_email_log(email_address, options = {})
+      endpoint = "customers/#{email_address}/logs"
+      params   = {}
+
+      params[:count]       = options[:count]       unless options[:count].nil?
+      params[:created_gt]  = options[:created_gt]  unless options[:created_gt].nil?
+      params[:created_lt]  = options[:created_lt]  unless options[:created_lt].nil?
+      if params.present?
```

Comment:
@kiragrom this looks like a RoR thing, which means we can't use it here. It's causing our Travis build to fail for your PR (https://travis-ci.org/sendwithus/sendwithus_ruby/builds/116131699)
If you can get this working without .present? and our build passes it looks good!